ls25/auto/admin.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib import admin
+from .models import Auto, ModelAuto, UserAuto, BrandAuto
+
+
+@admin.register(Auto)
+class AutoAdmin(admin.ModelAdmin):
+    list_display = ('model', 'vin_num', 'status', 'brand', 'auto_user')
+    list_filter = ('status', 'brand')
+    search_fields = ('vin_num', )
+
+
+@admin.register(ModelAuto)
+class ModelAutoAdmin(admin.ModelAdmin):
+    list_display = ('name', 'logo_brand_auto', 'count_auto')
+    list_filter = ('logo_brand_auto', )
```

Comment:
how can you filter by logo?

Reply:
hmm .. it s a FK.. rename this field.. if I call this field logo - I expected Image here